### PR TITLE
Vendor containers/storage v1.12.4

### PR DESCRIPTION
**Pull request by saschagrunert**
I updated the contribution guidelines about dependency management beside the vendoring.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -193,6 +193,10 @@ ${RELEASE_TOOL}:
 ${GOLANGCI_LINT}:
 	$(call go-build,golangci-lint,./vendor/github.com/golangci/golangci-lint/cmd/golangci-lint)
 
+vendor-update:
```
Comment (vrothberg) [inline Makefile]: I don't feel strongly about it, but I prefer to force all developers actually using `go` for that (mostly for educational purposes). Adding an alias for `go get -u` seems a bit redundant.

Comment (saschagrunert) [reply Makefile]: Hm, I think I will drop it and update the CONTRIBUTING.md guideline.

**Comment by rhatdan** [comment]
@saschagrunert I think we should wait for v1.12.4, since a big PR just got merged that was put in specifically for CRI-O.  I will create a new storage release today, so CRI-O can vendor it.

**Comment by saschagrunert** [comment]
> @saschagrunert I think we should wait for v1.12.4, since a big PR just got merged that was put in specifically for CRI-O. I will create a new storage release today, so CRI-O can vendor it.

Sounds good, let's wait for it.

/hold

**Comment by rhatdan** [comment]
https://github.com/containers/storage/releases/tag/v1.12.4 is released

**Comment by saschagrunert** [comment]
@rhatdan I updated to v1.12.4, PTAL.

**Comment by rhatdan** [comment]
LGTM
@umohnani8 @haircommander @mrunalp @vrothberg PTAL

**Pull request merged**